src/sdg/producten/admin/product.py

```diff
@@ -149,6 +155,12 @@ def is_referentie(self, obj):
 
     is_referentie.boolean = True
 
+    def _referentie_product(self, obj):
+        if obj.is_referentie_product:
+            return "—"
+        else:
+            return f"{obj.referentie_product} (referentie)"
```

Comment:
I get that this makes the change transparent, but since the column is named "referentie product" we can just remove the (referentie) part.
```suggestion
            return f"{obj.referentie_product}"
```